tests/e2e/utils/src/components.js

```diff
@@ -322,6 +322,30 @@ const createGroupedProduct = async (groupedProduct = defaultGroupedProduct) => {
 	return id;
 };
 
+/**
+ * Use the API to create an order with the provided details.
+ *
+ * @param {object} orderOptions
+ * @returns {Promise<number>} ID of the created order.
+ */
+const createOrder = async ( orderOptions = {} ) => {
+	const newOrder = {
+		...( orderOptions.status ) && { status: orderOptions.status },
+		...( orderOptions.customerId ) && { customer_id: orderOptions.customerId },
```

Comment:
Optional tweak- In the API package Order model customer ID is `customerId` and line items is `lineItems`.

Reply:
Thanks for the feedback @rrennick! I went to update it. And while `customerId` works as expected, changing `line_items` to `lineItems` doesn't seem to work (it fails to add the items to the order). So I think I will leave this as-is for now since this seems to require more investigating into why that isn't working as expected.